CONFIGDOC.md

```diff
@@ -78,9 +81,7 @@ You can specify build options for your MATLAB build by first selecting **Build o
 
 MATLAB exits with exit code 0 if the specified tasks run without error. Otherwise, MATLAB terminates with a nonzero exit code, which causes the step to fail.
 
-Starting in R2024a, you can view the results of running a MATLAB build in your Jenkins interface. After your build runs, the Jenkins build summary page displays the number of tasks that ran, failed, and were skipped. You can click the **MATLAB Build Results** link on the page to access the table of task results. The table provides information about each task that was part of the MATLAB build. Click a task name in the table to go to the relevant build log information on the **Console Output** page.
-
-![Table of MATLAB build results including three tasks. Each table row includes a clickable task name, its status, description, and duration.](https://github.com/user-attachments/assets/97fd7dc5-b00f-45bc-97a8-990ce26123d4)
+You can access the results of running a MATLAB build directly from your Jenkins interface. For more information, see [View MATLAB Build and Test Results](#view-matlab-build-and-test-results).

```

Comment:
If we add statements for both Run MATLAB Build and Run MATLAB Tests sections, we can link to specific sub-section of viewing results like `#view-build-results` instead of `#view-matlab-build-and-test-results`

Reply:
Makes sense.